.github/workflows/native-cron-build.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         run: mvn -B install -DskipTests -DskipITs -Dformat.skip
 
       - name: Run integration tests in native
-        run: mvn -B --settings .github/mvn-settings.xml verify -f integration-tests/pom.xml --fail-at-end -Dno-format -Ddocker -Dnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:20.1.0-java${{ matrix.java }} -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-db2 -Dtest-amazon-services -Dtest-vault -Dtest-neo4j -Dtest-keycloak -Dtest-kafka -Dtest-mssql -Dtest-mariadb -Dmariadb.url="jdbc:mariadb://localhost:3308/hibernate_orm_test" -pl '!io.quarkus:quarkus-integration-test-google-cloud-functions-http,!io.quarkus:quarkus-integration-test-google-cloud-functions,!integration-tests/funqy-google-cloud-function'
+        run: mvn -B --settings .github/mvn-settings.xml verify -f integration-tests/pom.xml --fail-at-end -Dno-format -Ddocker -Dnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:20.1.0-java${{ matrix.java }} -Dtest-postgresql -Dtest-elasticsearch -Dtest-mysql -Dtest-db2 -Dtest-amazon-services -Dtest-vault -Dtest-neo4j -Dtest-keycloak -Dtest-kafka -Dtest-mssql -Dtest-mariadb -Dmariadb.url="jdbc:mariadb://localhost:3308/hibernate_orm_test" -pl '!io.quarkus:quarkus-integration-test-google-cloud-functions-http,!io.quarkus:quarkus-integration-test-google-cloud-functions'
```

Comment:
Hum. I didn't see that but I really don't like the fact that we exclude these modules. We are supposed to support native everywhere.

@loicmathieu what's the reason of this?

Reply:
Google Cloud Functions only runs inside the Google Cloud Function Java runtime which is a Jetty server.
That's why it didn't runs in native.

Reply:
For me, there is no point of running a Function in native, this will no longuer be a function as we will need to include a runtime (wich one ?), create a bridge from the function API to this runtime, find a way to listen to the function event (PubSub, Storage, ...). 
So we will end up re-implementing the function runtime provided by Google ...

Reply:
If so couldn't we configure it in the pom of these modules instead (if configuration is needed but I suppose it will just build fine and won't build a native image)? I mean it's really no biggie to build these projects, it's really not something that will take time compared to building the other native images. And preferable to having to maintain this list IMHO.

Reply:
Azure functions didn't run on native neither ...

Reply:
Google Cloud Functions only works in Java 11. Native build is Java 8 that's why we exlude the ITs from the build.
Same is done on the standard CI, the Google Cloud Functions ITs are also excluded there.